package.json

```diff
@@ -33,7 +33,6 @@
   "devDependencies": {
     "chai": "^3.5.0",
     "eslint": "^2.12.0",
-    "babel-eslint": "^7.2.1",
```

Comment:
I'm sorry that I chose the wrong branch (master) to start a PR a few hours ago. Now I will choose Fix-checklist-issue branch to start a new pull request.